evm/src/executor/inspector/cheatcodes/expect.rs

```diff
@@ -214,6 +214,8 @@ pub struct MockCallDataContext {
     pub calldata: Bytes,
     /// The value to match for mock
     pub value: Option<U256>,
+    /// Reverts
+    pub reverts: bool,
```

Comment:
this is the marker for `mockCall` vs `mockCallRevert`?

Reply:
Yes that is correct, I shall add a better comment as well.